apps/site-app/src/containers/AddAccount/Restore/ConnectButton.tsx

```diff
@@ -1,25 +1,38 @@
 import { Form } from "semantic-ui-react"
 import React, { useCallback } from "react"
 import { ButtonPrimary } from "@thecointech/site-base/components/Buttons";
+import { FormattedMessage } from "react-intl";
 
 type Props = {
   disabled: boolean,
   loading: boolean,
   isVisible: boolean;
   onClick: () => void;
 }
-export const ConnectButton = (props: Props) => {
+
+const connectButton = { id:"app.addAccount.restore.connectButton.",
+                    defaultMessage:"Restore from Google",
+                    description:"The text for the create your password text"};
+
+
+export const ConnectButton : React.FC<Props> = (props)=> {
   const { isVisible, disabled, loading } = props;
 
   const onClick = useCallback((event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
     event?.preventDefault();
     props.onClick();
   }, [props.onClick]);
+  const linkToRestore = props.children 
```

Comment:
Added an option to have the image and the text I want or anything else

Reply:
This is a pretty elegant solution.

Reply:
The only suggestion I would make is to wrap this in it's own component.  Currently, you'll always create the components, even if you end up returning `null`.  If you wrapped it in it's own component it would only be constructed if necessary.

Reply:
Will do that in another PR